db/migrate/20170612163927_add_email_sent_at_to_rusers.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailSentAtToRusers < ActiveRecord::Migration
```

Comment:
And I think we can set this aside for the time being -- save it in a branch on your fork of the project and we can figure out if/how it fits in later. OK?